truss/templates/triton/config.pbtxt.jinja

```diff
@@ -0,0 +1,34 @@
+name: "model"
+backend: "python"
+max_batch_size: {{ max_batch_size }}
```

Comment:
should this be in the below if statement? Why have the batch size if not batching?